packages/components/src/external-link/index.tsx

```diff
@@ -9,13 +9,11 @@ import type { ForwardedRef } from 'react';
  */
 import { __ } from '@wordpress/i18n';
 import { forwardRef } from '@wordpress/element';
-import { external } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { VisuallyHidden } from '../visually-hidden';
-import { StyledIcon } from './styles/external-link-styles';
```

Comment:
I favor removal of Emotion when we can. 👍 

We already had classnames used here, so perhaps we should have used SCSS anyway.